includes/admin_import.php

```diff
@@ -213,7 +213,8 @@ function tsml_import_page()
                     tsml_delete_orphans();
 
                     if (count($change_log) && $tsml_debug) {
-                        $message = tsml_build_import_change_report($data_source_name, $change_log);
+                        $message = __('<h2>' . $data_source_name . __(' Feed Update Report', '12-step-meeting-list') . '</h2>');
+                        $message .= tsml_build_import_change_report($data_source_name, $change_log);
```

Comment:
do we need this change? it just adds a title for the table in debug mode - feel like it'll be obvious what this is because the user just clicked refresh

Reply:
I  like it because sometimes when there's lots of Data Sources, you might walk away for lunch or something and it runs while you're gone. Nice to know which report it is if you have lots of different data sources registered.

Reply:
How about a compromise here? Kill the title line on the debug report, but keep the header.